src/package.json

```diff
@@ -1,6 +1,6 @@
 {
-	"name": "nativescript-apple-sign-in",
-	"version": "1.1.0",
+	"name": "nativescript-apple-sign-in-enduco",
```

Comment:
you made some changes in this file that shouldn't go into the PR. all the references to enduco should be reverted. After doing that, i was able to run the demo app 👍

Reply:
Perfect, thanks!

Reply:
Yeah, please revert the ones in package.json as well. Feel free to add yourself to the authors section though.